index.html

```diff
@@ -13,10 +13,10 @@
     <script type="text/javascript" class="remove">
       var respecConfig = {
         // specification status (e.g. WD, LCWD, NOTE, etc.). If in doubt use ED.
-        specStatus: "CG-FINAL",
+        specStatus: "FPWD",
 
         // the specification's short name, as in http://www.w3.org/TR/short-name/
-        shortName: "claims-data-model",
+        shortName: "verifiable-claims-data-model",
```

Comment:
So, this depends on what @liamquin actually applied for, but I can see from the [22 May Minutes]() that it did, indeed, seem to be "vc-use-cases", at least that seems to be implied. I'll update both this PR, and the existing PR for use cases.